src/utils/ImportExportHelper.ts

```diff
@@ -0,0 +1,59 @@
+// Utility methods to call import/export Jahia API
+
+// http://localhost:8080/cms/export/default/export.zip?exportformat=site&live=true&users=true&path=/groups&path=/settings/mail-server-1&sitebox=digitall&exportPath=abc
+// Export
+
+import RequestOptions = Cypress.RequestOptions;
+import {JahiaServer} from '../support';
+
+const API_NAME = '/cms/export';
+
+export type ExportParameters = {
+    viewContent?: boolean;
+    viewVersion?: boolean;
+    viewAcl?: boolean;
+    viewLinks?: boolean;
+    viewMetadata?: boolean;
+    viewWorkflow?: boolean;
+    exportPath?: string;
+    exportformat: 'all' | 'site' | 'xml' | 'zip';
+    root?: string;
+    live?: boolean;
+    users?: boolean;
+    sitebox?: string[];
+    paths?: string[];
+    cleanup?: 'template' | 'simple';
+    filesToZip?: string;
+}
+
+const request = (workspace: string, nodePath: string, exportFormat: string, params: string, jahiaServer: JahiaServer): Partial<RequestOptions> => ({
```

Comment:
Please address the linter warning (note that if that if we believe that warning does not make sense, it can also be disabled

For example with, this at the top of the file:
```
/* eslint max-params: ["warning", 5] */
```

or inside the code
```
 // eslint-disable-next-line max-params
```

btw, using a config object also make sense, and is probably better than skipping the linter warning, for example here: https://github.com/Jahia/jahia-cypress/blob/7111d21a1b4741ab2f96dc2d2f4c0fbfbcf8b058/src/utils/SAMHelper.ts#L36

Reply:
Done